apps/asap-server/test/handlers/research-outputs/fetch.test.ts

```diff
@@ -0,0 +1,146 @@
+import Chance from 'chance';
+import nock from 'nock';
+import { APIGatewayProxyResult } from 'aws-lambda';
+import { config as authConfig } from '@asap-hub/auth';
+import { config } from '@asap-hub/services-common';
+
+import { identity } from '../../helpers/squidex';
+import { CMSResearchOutput } from '../../../src/entities/research-outputs';
+import { handler } from '../../../src/handlers/research-outputs/fetch';
+import { apiGatewayEvent } from '../../helpers/events';
+import { ResearchOutputResponse } from '@asap-hub/model';
+
+const chance = new Chance();
+describe('GET /research-outputs', () => {
+  beforeEach(() => {
+    nock.cleanAll();
+  });
+
+  afterEach(() => {
```

Comment:
Shouldn't need to do this twice

Reply:
I agree with you :( but we have some misbehaving test suits. It's on my todo list to address this :+1: